src/components/AdminSettings.vue

```diff
@@ -343,6 +349,19 @@ export default {
 				this.demoServers = []
 			}
 		},
+		checkNginx() {
+			try {
+				axios.get(generateUrl('/apps/richdocuments/settings/webserver')).then((response) => {
+					return response.data
+				}).then(data => {
+					if (data.status === 'success' && data.data.message.indexOf('nginx') >= 0) {
+						this.isNginx = true
+					}
+				})
```

Comment:
We could actually also test if apps/richdocumentscode/proxy.php?status is returning a proper 200 response without following redirects. Then we could only show the error when the status is not starting anymore, which would also solve the issue with a failing connection during loolwsd startup.

Reply:
I guess that will be in a different pr